tests/playwright-test/reporter-html.spec.ts

```diff
@@ -2757,8 +2764,12 @@ for (const useIntermediateMergeReport of [true, false] as const) {
         'uncommitted.txt': `uncommitted file`,
         'playwright.config.ts': `
           export default {
-            populateGitInfo: true,
-            metadata: { foo: 'value1', bar: { prop: 'value2' }, baz: ['value3', 123] }
+            metadata: {
+              'git.commit.info': {},
```

Comment:
Looks like we are deliberately making it less readable by requiring '' around the name.